extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/boot/FastBootMetadataBuilder.java

```diff
@@ -399,6 +400,12 @@ private MergedSettings mergeSettings(QuarkusPersistenceUnitDefinition puDefiniti
             }
         }
 
+        // If there's no user provided mapper but we have Jackson available,
+        // we want to add a "default" Quarkus mapper that delegates to the Quarkus-configured ObjectMapper:
+        if (!cfg.containsKey(AvailableSettings.JSON_FORMAT_MAPPER) && puDefinition.isJacksonPresent()) {
+            cfg.put(AvailableSettings.JSON_FORMAT_MAPPER, new QuarkusJsonFormatMapper());
+        }
```

Comment:
I didn't notice this in my earlier review.

I think there are multiple possible ways of serializing JSON (and perhaps XML) in Quarkus. I know there's support for json-p in particular.
Could you look if you can also make it work with other implementations than Jackson? See [here for JSON](https://quarkus.io/guides/rest-json#json), for XML I couldn't find anything with a quick search... I think there's only jaxb?